packages/react-form-state/package.json

```diff
@@ -27,7 +27,7 @@
     "tslib": "^1.9.3"
   },
   "peerDependencies": {
-    "react": "^16.4.1"
+    "react": "^16.8.1"
```

Comment:
Here we will need the higher peer dependency for a hooks rewrite.

Reply:
Yes but we should wait until we actually do the hook version to bump the peer dependency